test/java/seedu/addressbook/parser/UtilsTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.parser;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+public class UtilsTest {
+
+	@Test
+	public void isAnyNull() {
+		assertTrue(Utils.isAnyNull(new Integer(3), null));
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.